src/helpers/pivot/pivot_helpers.ts

```diff
@@ -14,6 +21,7 @@ import {
   PivotField,
   PivotTableCell,
 } from "../../types/pivot";
+import { domainToColRowDomain } from "./pivot_domain_helpers";
```

Comment:
circular dependency :/

Reply:
I think it make a bit more sense for `pivot_helpers` to be able to import `pivot_domain_helpers` ? I moved the constant `PREVIOUS_VALUE` inside `pivot_domain_helpers` to avoid havign somethign circular.